compiler/lexer.cpp

```diff
@@ -3323,17 +3289,7 @@ itoh(ucell val)
     static char itohstr[30];
     char* ptr;
     int i, nibble[16]; /* a 64-bit hexadecimal cell has 16 nibbles */
-    int max;
-
-#if PAWN_CELL_SIZE == 16
-    max = 4;
-#elif PAWN_CELL_SIZE == 32
-    max = 8;
-#elif PAWN_CELL_SIZE == 64
-    max = 16;
-#else
-#    error Unsupported cell size
-#endif
+    int max = 8;

```

Comment:
Oh god, dvander.  How can you waste all that memory in the `nibble` array like that?